Just for the case you plan to synch your repo

mcm1957 opened the pull request.

Apollon77 commented:
Thx. No I rather need to cleanup the repo becaus it is nopt moved to community ... also my fork is wring then. I will delete it soon

mcm1957 committed:
Merge pull request #145 from iobroker-community-adapters/dependabot/npm_and_yarn/qs-6.12.1

Bump qs from 6.11.2 to 6.12.1

mcm1957 committed:
chore: release v1.4.0

- (mcm1957) Adapter requires node.js >= 18 and js-controller >= 5 now
- (mcm1957) Dependencies have been updated

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.1.0 to 3.1.4 (#149)

dependabot[bot] committed:
Bump tough-cookie from 4.1.3 to 4.1.4 (#150)

mcm1957 committed:
move german docu to seperated README_de.md

mcm1957 committed:
Create CHANGELOG_OLD.md

mcm1957 committed:
added translations

dependabot[bot] committed:
Bump @types/node from 20.12.7 to 20.13.0 (#156)

dependabot[bot] committed:
Bump chai-as-promised from 7.1.1 to 7.1.2 (#161)

dependabot[bot] committed:
Bump typescript from 5.4.5 to 5.5.2 (#163)

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 3.7.0 to 3.7.2 (#164)

dependabot[bot] committed:
Bump @types/node from 20.13.0 to 20.14.9 (#165)

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.1.4 to 3.1.6 (#167)

dependabot[bot] committed:
Bump prettier from 3.2.5 to 3.3.2 (#169)

foxriver76 committed:
fixed invalid min/max values

foxriver76 committed:
chore: release v1.4.1

* (foxriver76) fixed invalid min/max values

dependabot[bot] committed:
Bump eslint-plugin-prettier from 5.1.3 to 5.2.1 (#175)

dependabot[bot] committed:
Bump typescript from 5.5.2 to 5.5.4 (#179)

dependabot[bot] committed:
Bump @alcalzone/release-script from 3.7.0 to 3.8.0 (#183)

dependabot[bot] committed:
Bump chai from 4.4.1 to 4.5.0 (#178)

dependabot[bot] committed:
Bump prettier from 3.3.2 to 3.3.3 (#177)

dependabot[bot] committed:
Bump typescript from 5.5.4 to 5.6.2 (#196)

TA2k committed:
catch set api value

TA2k committed:
Merge branch 'master' of https://github.com/iobroker-community-adapters/ioBroker.homeconnect

TA2k committed:
update dep

TA2k committed:
fix lints

TA2k committed:
stringify api value

TA2k committed:
add conditional stringify

TA2k committed:
chore: release v1.4.2

- fix for devices with object values

dependabot[bot] committed:
Bump @types/node from 22.7.9 to 22.8.6 (#207)

simatec committed:
(simatec) Responsive Design added

TA2k committed:
fix haid for get device

TA2k committed:
Merge branch 'master' of https://github.com/iobroker-community-adapters/ioBroker.homeconnect

TA2k committed:
fix lock file

mcm1957 committed:
Update README.md

mcm1957 committed:
Update io-package.json

mcm1957 committed:
Update package.json

mcm1957 committed:
Delete .npmignore

TA2k committed:
chore: release v1.4.3

- fix for -001 devices
- (simatec) Adapter has been adapted to meet Responsive Design rules.

mcm1957 committed:
Update README.md

dependabot[bot] committed:
bump @eslint/eslintrc from 3.1.0 to 3.2.0 (#212)

dependabot[bot] committed:
bump axios from 1.7.7 to 1.7.8 (#213)

dependabot[bot] committed:
bump qs from 6.13.0 to 6.13.1 (#215)

dependabot[bot] committed:
bump prettier from 3.3.3 to 3.4.1 (#216)

dependabot[bot] committed:
bump @types/node from 22.9.0 to 22.10.1 (#218)

dependabot[bot] committed:
bump eslint from 9.14.0 to 9.16.0 (#214)

dependabot[bot] committed:
bump typescript from 5.6.3 to 5.7.2 (#219)

dependabot[bot] committed:
Bump @types/node from 22.10.1 to 22.10.3 (#221)

dependabot[bot] committed:
Bump axios from 1.7.8 to 1.7.9 (#222)

dependabot[bot] committed:
Bump globals from 15.12.0 to 15.14.0 (#223)

dependabot[bot] committed:
Bump eslint from 9.16.0 to 9.17.0 (#224)

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.2.2 to 3.2.3 (#227)

dependabot[bot] committed:
Bump http-cookie-agent from 6.0.6 to 6.0.8 (#226)

dependabot[bot] committed:
Bump prettier from 3.4.1 to 3.4.2 (#225)

dependabot[bot] committed:
Bump eslint from 9.17.0 to 9.19.0 (#232)

dependabot[bot] committed:
Bump typescript from 5.7.2 to 5.7.3 (#233)

dependabot[bot] committed:
Bump @iobroker/testing from 5.0.0 to 5.0.3 (#234)

dependabot[bot] committed:
Bump eslint-plugin-prettier from 5.2.1 to 5.2.3 (#236)

dependabot[bot] committed:
Bump @types/node from 22.10.3 to 22.13.0 (#239)

dependabot[bot] committed:
Bump eslint from 9.19.0 to 9.21.0 (#240)

dependabot[bot] committed:
Bump typescript from 5.7.3 to 5.8.2 (#245)

dependabot[bot] committed:
Bump @types/node from 22.13.0 to 22.13.8 (#250)

dependabot[bot] committed:
Bump prettier from 3.4.2 to 3.5.2 (#243)

dependabot[bot] committed:
Bump @eslint/eslintrc from 3.3.0 to 3.3.1 (#258)

dependabot[bot] committed:
Bump eslint-plugin-prettier from 5.2.3 to 5.2.5 (#260)

dependabot[bot] committed:
Bump eslint from 9.21.0 to 9.23.0 (#262)

dependabot[bot] committed:
Bump @tsconfig/node14 from 14.1.2 to 14.1.3 (#255)

dependabot[bot] committed:
Bump @types/node from 22.13.8 to 22.13.14 (#257)

dependabot[bot] committed:
Bump @iobroker/testing from 5.0.3 to 5.0.4 (#263)

dependabot[bot] committed:
Bump prettier from 3.5.2 to 3.5.3 (#261)

dependabot[bot] committed:
Bump eslint from 9.23.0 to 9.25.1 (#267)

dependabot[bot] committed:
Bump typescript from 5.8.2 to 5.8.3 (#269)

dependabot[bot] committed:
Bump @types/node from 22.13.17 to 22.15.3 (#272)

dependabot[bot] committed:
Bump eslint-plugin-prettier from 5.2.5 to 5.2.6 (#268)

dependabot[bot] committed:
Bump eslint from 9.25.1 to 9.28.0 (#274)

dependabot[bot] committed:
Bump eslint-plugin-prettier from 5.2.6 to 5.4.1 (#276)

dependabot[bot] committed:
Bump @types/node from 22.15.3 to 22.15.29 (#280)

dependabot[bot] committed:
Bump @types/node from 22.15.29 to 22.15.30 (#281)

mcm1957 committed:
Update test-and-release.yml . add node 24 tests, drop node 18

mcm1957 committed:
Update package.json - require node 20

mcm1957 committed:
Update README.md

Lucky-ESA committed:
code and objects clean up

Lucky-ESA committed:
changed to jsonConfig

Lucky-ESA committed:
update schema

Lucky-ESA committed:
admin & controller required & docs clean up

Lucky-ESA committed:
update dependencies

Lucky-ESA committed:
update

Lucky-ESA committed:
license adjusted

dependabot[bot] committed:
Bump axios from 1.9.0 to 1.10.0

Bumps [axios](https://github.com/axios/axios) from 1.9.0 to 1.10.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.9.0...v1.10.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.10.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @tsconfig/node20 from 20.1.5 to 20.1.6 (#292)

dependabot[bot] committed:
Bump mocha from 11.6.0 to 11.7.0 (#290)

dependabot[bot] committed:
Bump mocha from 11.7.0 to 11.7.1 (#294)

dependabot[bot] committed:
Bump sinon from 20.0.0 to 21.0.0

Bumps [sinon](https://github.com/sinonjs/sinon) from 20.0.0 to 21.0.0.
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/commits)

---
updated-dependencies:
- dependency-name: sinon
  dependency-version: 21.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Merge pull request #286 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.10.0

Bump axios from 1.9.0 to 1.10.0

mcm1957 committed:
Merge pull request #288 from iobroker-community-adapters/dependabot/npm_and_yarn/sinon-21.0.0

Bump sinon from 20.0.0 to 21.0.0

dependabot[bot] committed:
Bump @types/node from 22.15.31 to 24.1.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.15.31 to 24.1.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 24.1.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump http-cookie-agent from 7.0.1 to 7.0.2 (#297)

dependabot[bot] committed:
Bump axios from 1.10.0 to 1.11.0

Bumps [axios](https://github.com/axios/axios) from 1.10.0 to 1.11.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.10.0...v1.11.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.11.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
fix object attribut & check device

Lucky-ESA committed:
add rate limiting

Lucky-ESA committed:
admin 7.6.17 required

Lucky-ESA committed:
catch rate limit states

Lucky-ESA committed:
fix day counter reset

Lucky-ESA committed:
add response type

Lucky-ESA committed:
fix value & loglevel changed

Lucky-ESA committed:
Merge pull request #300 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.11.0

Bump axios from 1.10.0 to 1.11.0

Lucky-ESA committed:
Merge pull request #299 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-24.1.0

Bump @types/node from 22.15.31 to 24.1.0

Lucky-ESA committed:
update devDeps

dependabot[bot] committed:
Bump typescript from 5.8.3 to 5.9.2 (#301)

dependabot[bot] committed:
Bump @iobroker/testing from 5.1.0 to 5.1.1 (#305)

dependabot[bot] committed:
Bump actions/checkout from 4 to 5

Bumps [actions/checkout](https://github.com/actions/checkout) from 4 to 5.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #306 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-5

Bump actions/checkout from 4 to 5

Lucky-ESA committed:
update package-lock

Lucky-ESA committed:
chore: release v1.5.0

- (Lucky-ESA) Clean up state roles and code
- (Lucky-ESA) Added rate limiting
- (Lucky-ESA) Dependencies updated
- (Lucky-ESA) Added language selection
- (Lucky-ESA) Migrated to ESLint 9
- (Lucky-ESA) Adapter requires js-controller >= 6.0.11 now
- (Lucky-ESA) Adapter requires admin >= 7.6.17 now
- (mcm1957) Adapter requires node.js >= 20 now

Lucky-ESA committed:
fix name of the objects are deleted

Lucky-ESA committed:
change schema

Lucky-ESA committed:
update deps

Lucky-ESA committed:
update

Lucky-ESA committed:
chore: release v1.5.1

- (Lucky-ESA) Fixed: Name of the objects are deleted

Copilot committed:
Initial plan

Copilot committed:
feat: implement complete GitHub Copilot setup for ioBroker homeconnect adapter

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #311 from iobroker-community-adapters/copilot/fix-9a84e39c-d00a-4689-ba31-c52996bd1b06

feat: setup GitHub Copilot instructions for ioBroker homeconnect adapter

dependabot[bot] committed:
bump @types/node from 24.5.2 to 24.6.1

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 24.6.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump @iobroker/eslint-config from 2.1.0 to 2.2.0

---
updated-dependencies:
- dependency-name: "@iobroker/eslint-config"
  dependency-version: 2.2.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #314 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-24.6.1

bump @types/node from 24.5.2 to 24.6.1

Lucky-ESA committed:
Merge pull request #315 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/eslint-config-2.2.0

bump @iobroker/eslint-config from 2.1.0 to 2.2.0

dependabot[bot] committed:
bump @iobroker/adapter-dev from 1.4.0 to 1.5.0

---
updated-dependencies:
- dependency-name: "@iobroker/adapter-dev"
  dependency-version: 1.5.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #316 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/adapter-dev-1.5.0

bump @iobroker/adapter-dev from 1.4.0 to 1.5.0

dependabot[bot] committed:
bump typescript from 5.9.2 to 5.9.3

---
updated-dependencies:
- dependency-name: typescript
  dependency-version: 5.9.3
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
license changed

Lucky-ESA committed:
Merge pull request #317 from iobroker-community-adapters/dependabot/npm_and_yarn/typescript-5.9.3

bump typescript from 5.9.2 to 5.9.3

github-actions[bot] committed:
Update from template: X0000-MigrateToTrustedPublishing

Lucky-ESA committed:
slow uodate & fix object states

Lucky-ESA committed:
Merge pull request #325 from ioBroker-Bot/update-from-template-X0000-MigrateToTrustedPublishing-1760790433

[iobroker-bot] Migrate to NPM Trusted Publishing

Copilot committed:
Initial plan

dependabot[bot] committed:
bump actions/checkout from 4 to 5

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump actions/github-script from 7 to 8

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script from 3.8.0 to 5.0.0

Bumps [@alcalzone/release-script](https://github.com/AlCalzone/release-script) from 3.8.0 to 5.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.8.0...v5.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script"
  dependency-version: 5.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-license from 3.7.0 to 4.0.0

Bumps [@alcalzone/release-script-plugin-license](https://github.com/AlCalzone/release-script) from 3.7.0 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.0...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-license"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.6.1 to 24.9.1 (#329)

Copilot committed:
Update ioBroker Copilot Instructions template from v0.4.0 to v0.4.2

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #313 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-5

bump actions/checkout from 4 to 5

mcm1957 committed:
Merge pull request #312 from iobroker-community-adapters/dependabot/github_actions/actions/github-script-8

bump actions/github-script from 7 to 8

mcm1957 committed:
Merge pull request #330 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-5.0.0

Bump @alcalzone/release-script from 3.8.0 to 5.0.0

mcm1957 committed:
Merge pull request #331 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-license-4.0.0

Bump @alcalzone/release-script-plugin-license from 3.7.0 to 4.0.0

mcm1957 committed:
Merge pull request #328 from iobroker-community-adapters/copilot/update-iobroker-template

Update ioBroker Copilot Instructions template from v0.4.0 to v0.4.2

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 3.7.2 to 4.0.0

Bumps [@alcalzone/release-script-plugin-iobroker](https://github.com/AlCalzone/release-script) from 3.7.2 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.2...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-iobroker"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-manual-review from 3.7.0 to 4.0.0

Bumps [@alcalzone/release-script-plugin-manual-review](https://github.com/AlCalzone/release-script) from 3.7.0 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.0...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-manual-review"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Merge pull request #332 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-iobroker-4.0.0

Bump @alcalzone/release-script-plugin-iobroker from 3.7.2 to 4.0.0

mcm1957 committed:
Merge pull request #333 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-manual-review-4.0.0

Bump @alcalzone/release-script-plugin-manual-review from 3.7.0 to 4.0.0

dependabot[bot] committed:
Bump @types/node from 24.9.1 to 24.9.2 (#334)

github-actions[bot] committed:
Update from template: X0000-randomDependabot

mcm1957 committed:
Merge pull request #337 from ioBroker-Bot/update-from-template-X0000-randomDependabot-1762825897

[iobroker-bot] Optimize Dependabot Schedule Configuration

dependabot[bot] committed:
Bump @tsconfig/node20 from 20.1.6 to 20.1.8 (#338)

dependabot[bot] committed:
Bump http-cookie-agent from 7.0.2 to 7.0.3 (#339)

dependabot[bot] committed:
Bump @iobroker/testing from 5.1.1 to 5.2.2 (#342)

dependabot[bot] committed:
Bump axios from 1.12.2 to 1.13.2

Bumps [axios](https://github.com/axios/axios) from 1.12.2 to 1.13.2.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.12.2...v1.13.2)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.13.2
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.9.2 to 24.10.1 (#340)

dependabot[bot] committed:
Bump actions/checkout from 5 to 6

Bumps [actions/checkout](https://github.com/actions/checkout) from 5 to 6.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump eventsource from 4.0.0 to 4.1.0

Bumps [eventsource](https://github.com/EventSource/eventsource) from 4.0.0 to 4.1.0.
- [Release notes](https://github.com/EventSource/eventsource/releases)
- [Changelog](https://github.com/EventSource/eventsource/blob/main/CHANGELOG.md)
- [Commits](https://github.com/EventSource/eventsource/compare/v4.0.0...v4.1.0)

---
updated-dependencies:
- dependency-name: eventsource
  dependency-version: 4.1.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #341 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.13.2

Bump axios from 1.12.2 to 1.13.2

Lucky-ESA committed:
Merge pull request #343 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-6

Bump actions/checkout from 5 to 6

Lucky-ESA committed:
Merge pull request #344 from iobroker-community-adapters/dependabot/npm_and_yarn/eventsource-4.1.0

Bump eventsource from 4.0.0 to 4.1.0

Lucky-ESA committed:
own request added

Lucky-ESA committed:
update dependencies

Lucky-ESA committed:
catch start ratelimit

Lucky-ESA committed:
update

Lucky-ESA committed:
chore: release v1.5.2

- (Lucky-ESA) Rate limit of 50 requests per minute intercepted at adapter startup
- (Lucky-ESA) Added custom request

github-actions[bot] committed:
Update from template: E40xx-fixSchemaLinkForVsCode

github-actions[bot] committed:
Update from template: X0000-prepareCommunityAdapters

mcm1957 committed:
Merge pull request #349 from ioBroker-Bot/update-from-template-X0000-prepareCommunityAdapters-1767097954

[iobroker-bot] General update for iobroker-community-adapters repository

github-actions[bot] committed:
Update from template: E6005-fixCopyrightYear

mcm1957 committed:
Merge pull request #346 from ioBroker-Bot/update-from-template-E40xx-fixSchemaLinkForVsCode-1766276783

[iobroker-bot] Fix Schema Links for VSCode

mcm1957 committed:
Merge pull request #350 from ioBroker-Bot/update-from-template-E6005-fixCopyrightYear-1767192945

[iobroker-bot] Fix Copyright Year

dependabot[bot] committed:
Bump qs from 6.14.0 to 6.14.1 (#352)

Lucky-ESA committed:
Fixed adapter crash if uri is empty

Lucky-ESA committed:
Fixed adapter crash if uri is empty

github-actions[bot] committed:
Update from template: X0000-setupAutomergeDependabot

mcm1957 committed:
Merge pull request #354 from ioBroker-Bot/update-from-template-X0000-setupAutomergeDependabot-1768515130

[iobroker-bot] Setup Auto-Merge for Dependabot PRs

github-actions[bot] committed:
Update from template: X0000-prepareCommunityAdapters

Copilot committed:
Initial plan

Copilot committed:
chore: update copilot-instructions.md from v0.4.2 to v0.5.7

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #358 from iobroker-community-adapters/copilot/update-iobroker-copilot-template

Update ioBroker Copilot Instructions template from v0.4.2 to v0.5.7

github-actions[bot] committed:
Update from template: X0000-updateNodeJsAtTestAndRelease

mcm1957 committed:
Merge pull request #360 from ioBroker-Bot/update-from-template-X0000-updateNodeJsAtTestAndRelease-1772718755

[iobroker-bot] Update Node.js versions in test-and-release workflow

github-actions[bot] committed:
Update from template: X0000-randomDependabot

mcm1957 committed:
Merge pull request #362 from ioBroker-Bot/update-from-template-X0000-randomDependabot-1773582619

[iobroker-bot] Optimize Dependabot Schedule Configuration

mcm1957 committed:
Merge pull request #356 from ioBroker-Bot/update-from-template-X0000-prepareCommunityAdapters-1771851220

[iobroker-bot] General update for iobroker-community-adapters repository

dependabot[bot] committed:
Bump axios from 1.13.2 to 1.13.6

Bumps [axios](https://github.com/axios/axios) from 1.13.2 to 1.13.6.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.13.2...v1.13.6)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.13.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script from 5.0.0 to 5.1.1

Bumps [@alcalzone/release-script](https://github.com/AlCalzone/release-script) from 5.0.0 to 5.1.1.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.0.0...v5.1.1)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script"
  dependency-version: 5.1.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump tough-cookie from 6.0.0 to 6.0.1

Bumps [tough-cookie](https://github.com/salesforce/tough-cookie) from 6.0.0 to 6.0.1.
- [Release notes](https://github.com/salesforce/tough-cookie/releases)
- [Changelog](https://github.com/salesforce/tough-cookie/blob/master/CHANGELOG.md)
- [Commits](https://github.com/salesforce/tough-cookie/compare/v6.0.0...v6.0.1)

---
updated-dependencies:
- dependency-name: tough-cookie
  dependency-version: 6.0.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @tsconfig/node20 from 20.1.8 to 20.1.9

Bumps [@tsconfig/node20](https://github.com/tsconfig/bases/tree/HEAD/bases) from 20.1.8 to 20.1.9.
- [Commits](https://github.com/tsconfig/bases/commits/HEAD/bases)

---
updated-dependencies:
- dependency-name: "@tsconfig/node20"
  dependency-version: 20.1.9
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #371 from iobroker-community-adapters/dependabot/npm_and_yarn/tough-cookie-6.0.1

Bump tough-cookie from 6.0.0 to 6.0.1

github-actions[bot] committed:
Merge pull request #368 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.13.6

Bump axios from 1.13.2 to 1.13.6

github-actions[bot] committed:
Merge pull request #372 from iobroker-community-adapters/dependabot/npm_and_yarn/tsconfig/node20-20.1.9

Bump @tsconfig/node20 from 20.1.8 to 20.1.9

github-actions[bot] committed:
Merge pull request #370 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-5.1.1

Bump @alcalzone/release-script from 5.0.0 to 5.1.1

dependabot[bot] committed:
Bump @types/node from 24.10.4 to 24.12.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 24.10.4 to 24.12.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 24.12.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-manual-review from 4.0.0 to 5.1.1

Bumps [@alcalzone/release-script-plugin-manual-review](https://github.com/AlCalzone/release-script) from 4.0.0 to 5.1.1.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v4.0.0...v5.1.1)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-manual-review"
  dependency-version: 5.1.1
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 4.0.0 to 5.1.2

Bumps [@alcalzone/release-script-plugin-iobroker](https://github.com/AlCalzone/release-script) from 4.0.0 to 5.1.2.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v4.0.0...v5.1.2)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-iobroker"
  dependency-version: 5.1.2
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #363 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-24.12.0

Bump @types/node from 24.10.4 to 24.12.0

github-actions[bot] committed:
Update from template: X0000-updateDependabotSettings

mcm1957 committed:
Merge pull request #374 from ioBroker-Bot/update-from-template-X0000-updateDependabotSettings-1775004063

[iobroker-bot] Update Dependabot Configuration – Add npm Cooldown

github-actions[bot] committed:
Update from template: X0000-updateNodeJsAtTestAndRelease

mcm1957 committed:
Merge pull request #379 from ioBroker-Bot/update-from-template-X0000-updateNodeJsAtTestAndRelease-1775507053

[iobroker-bot] Update Node.js versions in test-and-release workflow

mcm1957 committed:
Merge pull request #369 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-iobroker-5.1.2

Bump @alcalzone/release-script-plugin-iobroker from 4.0.0 to 5.1.2

mcm1957 committed:
Merge pull request #364 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-manual-review-5.1.1

Bump @alcalzone/release-script-plugin-manual-review from 4.0.0 to 5.1.1

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-license from 4.0.0 to 5.1.1

Bumps [@alcalzone/release-script-plugin-license](https://github.com/AlCalzone/release-script) from 4.0.0 to 5.1.1.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v4.0.0...v5.1.1)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-license"
  dependency-version: 5.1.1
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/github-script from 8 to 9

Bumps [actions/github-script](https://github.com/actions/github-script) from 8 to 9.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v8...v9)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '9'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.12.0 to 24.12.2

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 24.12.0 to 24.12.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 24.12.2
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump axios from 1.13.6 to 1.15.0

Bumps [axios](https://github.com/axios/axios) from 1.13.6 to 1.15.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.13.6...v1.15.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.15.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Merge pull request #366 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-license-5.1.1

Bump @alcalzone/release-script-plugin-license from 4.0.0 to 5.1.1

mcm1957 committed:
Merge pull request #381 from iobroker-community-adapters/dependabot/github_actions/actions/github-script-9

Bump actions/github-script from 8 to 9

github-actions[bot] committed:
Merge pull request #382 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-24.12.2

Bump @types/node from 24.12.0 to 24.12.2

dependabot[bot] committed:
Bump qs from 6.14.1 to 6.15.1

Bumps [qs](https://github.com/ljharb/qs) from 6.14.1 to 6.15.1.
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.14.1...v6.15.1)

---
updated-dependencies:
- dependency-name: qs
  dependency-version: 6.15.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #380 from iobroker-community-adapters/dependabot/npm_and_yarn/qs-6.15.1

Bump qs from 6.14.1 to 6.15.1

Lucky-ESA committed:
Merge pull request #383 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.15.0

Bump axios from 1.13.6 to 1.15.0

Lucky-ESA committed:
save value in active folder

github-actions[bot] committed:
Update from template: W8917-dependabot-addIgnoreTypesNode

github-actions[bot] committed:
Update from template: X0000-dropNode20

github-actions[bot] committed:
Update from template: S6020-addChangelogOld

dependabot[bot] committed:
Bump axios from 1.15.0 to 1.15.2

Bumps [axios](https://github.com/axios/axios) from 1.15.0 to 1.15.2.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.15.0...v1.15.2)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.15.2
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #394 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.15.2

Bump axios from 1.15.0 to 1.15.2

Lucky-ESA committed:
controlled adapter start

Lucky-ESA committed:
Merge pull request #389 from ioBroker-Bot/update-from-template-W8917-dependabot-addIgnoreTypesNode-1777667104

[iobroker-bot] Add Dependabot Ignore Rule for @types/node Major Version Updates

dependabot[bot] committed:
Bump axios from 1.15.2 to 1.16.0

Bumps [axios](https://github.com/axios/axios) from 1.15.2 to 1.16.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.15.2...v1.16.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.16.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge branch 'master' into update-from-template-X0000-dropNode20-1777716953

Lucky-ESA committed:
Merge pull request #392 from ioBroker-Bot/update-from-template-S6020-addChangelogOld-1777810551

[iobroker-bot] Add CHANGELOG_OLD.md to store older changelog entries

Lucky-ESA committed:
Merge pull request #391 from ioBroker-Bot/update-from-template-X0000-dropNode20-1777716953

[iobroker-bot] Drop Node.js 20 support (End of Life)

Lucky-ESA committed:
Merge pull request #395 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.16.0

Bump axios from 1.15.2 to 1.16.0

dependabot[bot] committed:
Bump axios-rate-limit from 1.4.0 to 1.9.0

Bumps [axios-rate-limit](https://github.com/aishek/axios-rate-limit) from 1.4.0 to 1.9.0.
- [Release notes](https://github.com/aishek/axios-rate-limit/releases)
- [Changelog](https://github.com/aishek/axios-rate-limit/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aishek/axios-rate-limit/compare/1.4.0...1.9.0)

---
updated-dependencies:
- dependency-name: axios-rate-limit
  dependency-version: 1.9.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Lucky-ESA committed:
Merge pull request #384 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-rate-limit-1.9.0

Bump axios-rate-limit from 1.4.0 to 1.9.0

mcm1957 committed:
Update Node version requirements in README

mcm1957 committed:
Delete .github/workflows/dependabot-auto-merge.yml.OLD

mcm1957 committed:
Implement concurrency for PR/branch workflows

Add concurrency settings to cancel previous runs on new commits.

mcm1957 committed:
chore: release v1.6.0

- (copilot) Adapter requires node.js >= 22 now
- (copilot) Adapter requires admin >= 7.7.22 now
- (Lucky-ESA) Fixed adapter crash if URI is empty
- (Lucky-ESA) Save remaining time in active folder
- (Lucky-ESA) Device monitoring starts only after the adapter has started (this may take up to 2 minutes)

TA2k committed:
cleanup relogin/login flow

TA2k committed:
improve log

TA2k committed:
chore: release v1.6.1

- Login/Refresh flow improved

mcm1957 committed:
Update README.md

Lucky-ESA committed:
added verificationUrl object

Lucky-ESA committed:
Fixed unnecessary object overwriting

Lucky-ESA committed:
delete logs

github-actions[bot] committed:
Update from template: X0000-setupAutomergeIobrokerBot

mcm1957 committed:
Merge pull request #399 from ioBroker-Bot/update-from-template-X0000-setupAutomergeIobrokerBot-1778756926

[iobroker-bot] Setup Auto-Merge for ioBroker-Bot PRs

github-actions[bot] committed:
Update from template: X0000-setupAutomergeIobrokerBot

mcm1957 committed:
Merge pull request #400 from ioBroker-Bot/update-from-template-X0000-setupAutomergeIobrokerBot-1778798834

[iobroker-bot] Setup Auto-Merge for ioBroker-Bot PRs

github-actions[bot] committed:
Update from template: S6020-addChangelogOld

github-actions[bot] committed:
Update from template: X0000-updateTypesNode

mcm1957 committed:
Merge pull request #401 from ioBroker-Bot/update-from-template-S6020-addChangelogOld-1778870396

[iobroker-bot] Add CHANGELOG_OLD.md to store older changelog entries

mcm1957 committed:
Merge pull request #403 from ioBroker-Bot/update-from-template-X0000-updateTypesNode-1779352734

[iobroker-bot] Update @types/node to match minimum supported Node.js version
